GomokuServer/src/GomokurServer.Core/Games/Entities/GameWithTimeControl.cs

```diff
@@ -46,44 +76,59 @@ public override GameTilePlacementResult PlaceTile(Tile tile, string playerId)
 			{
 				_blackClock.Stop();
 				_whiteClock.Stop();
-
 				return tilePlacementResult;
 			}
 
-			var (currentPlayerClock, opponentsClock) = currentColor == TileColor.Black
-				? (_blackClock, _whiteClock)
-				: (_whiteClock, _blackClock);
-
-			if (MovesHistory.Count == 0)
+			if (MovesHistory.Count == 2)
```

Comment:
What do you think, if we avoid this _magic number_ "2" ? And intead place it in constant const countGameInPlay ?

Reply:
Added IsTimeControlInitiated property